gnarails.gemspec

```
@@ -24,7 +24,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "rails", "~> 5.2.1"
   spec.add_dependency "thor"
 
-  spec.add_development_dependency "bundler", "~> 1.16"
+  spec.add_development_dependency "bundler", "~> 2.0.1"
```

Unfortunately, the commit (ba385b104f92bb5538ffff422254853cc7abf4bf) this pessimistic lock was added in doesn't have sufficient context around this particular change. However, I don't think we want to or need to enforce that people use bundler 2 to use this, and I'll take it on faith that there's a reason 1.15 and below is excluded. Will this work?

```suggestion
  spec.add_development_dependency "bundler", ">= 1.16", "< 3.0"
```